editor/tasks/leiningen/util/http_cache.clj

```diff
@@ -128,6 +128,10 @@
               (doto (File. tmp)
                 (.renameTo dst))
               dst))
+      301 (let [headers (:headers resp)
```

Comment:
The editor still looks for files in the old location. ie. d.defold.com/archive/sha1/* but these will now only contain redirects/symlinks. We need to handle these and download from the actual location.